README.md

```diff
@@ -109,34 +151,35 @@ Inside, make sure the following is there:
 
 exec openbox-session #-> This is the one that launches Openbox ;)
 ```
-10. Run "spotifypod.py" with autostart
+
+11. Run "spotifypod.py" with autostart
 
 `sudo nano /etc/xdg/openbox/autostart`
 
 
 and add the following command to launch spotifypod.py:
 
-```
-cd /home/pi/fork/retro-ipod-spotify-client/frontend/
+```bash
+cd /home/pi/retro-ipod-spotify-client/frontend/
 
-sudo -H -u pi python3 spotifypod.py &
+sudo -H -u pi --preserve-env=SPOTIPY_REDIRECT_URI,SPOTIPY_CLIENT_ID,SPOTIPY_CLIENT_SECRET python3 spotifypod.py &
 
-sudo /home/pi/fork/retro-ipod-spotify-client/clickwheel/click &
+sudo /home/pi/retro-ipod-spotify-client/clickwheel/click &
 ```
 
 _Make sure that the paths are ok with your setup!!_
 
 in ` sudo nano /etc/xdg/openbox/environment` all the variables needed to run spotifypod.py are set( SPOTIPY_CLIENT_ID, SPOTIPY_CLIENT_SECRET,SPOTIPY_REDIRECT_URI)
 
-```
+```bash
 export SPOTIPY_CLIENT_ID='your_SPOTIPY_CLIENT_ID'
 
 export SPOTIPY_CLIENT_SECRET='your_SPOTIPY_CLIENT_SECRET'
 
 export SPOTIPY_REDIRECT_URI='your_SPOTIPY_REDIRECT_URI'
 ```
```

Comment:
#41 @nato2k do we need those environment variables even if we added them in our `.bash_profile`file in step 9?